packages/tooltip/src/Tooltip.story.tsx

```diff
@@ -20,8 +20,21 @@ export default {
   },
 };
 
-const Template: ComponentStory<typeof Tooltip> = args => (
-  <Tooltip trigger={<Button>trigger</Button>} {...args} />
+const Template: ComponentStory<typeof Tooltip> = ({
+  darkMode,
+  ...args
+}: TooltipProps) => (
+  <div
+    className={css`
+      padding: 100px;
+    `}
+  >
+    <Tooltip
+      darkMode={darkMode}
```

Comment:
Small thing, but it looks like `darkMode` doesn't have to be broken out of the spread any more.

Reply:
I did this so that i can pass darkMode to the trigger button